### ISSUE TITLE
LOG-1505: SQS3script.py bug

### ISSUE THREAD

**Issue by so0k** [loggly/install-script#89]
Some users may have multiple AWS profiles, which causes their credential file to look as follows:

```ini
[default]
aws_access_key_id = MY_DEFAULT_ACCOUNT
aws_secret_access_key = MY_DEFAULT_SECRET

[personal]
aws_access_key_id = MY_PERSONAL_ACCOUNT
aws_secret_access_key = MY_PERSONAL_SECRET
```

the [following lines](https://github.com/loggly/install-script/blob/ea6395fc94f2cdf8da009c73be43c896773b48f6/AWSscripts/SQS3script.py#L54-L59) may cause "Unauthorized" errors while running the script:

```python
with open(os.environ['HOME'] + '/.aws/credentials') as f:
    for line in f:
        if "aws_access_key_id" in line:
             access_key = line.split("=",1)[1].strip()
        if "aws_secret_access_key" in line:
             secret_key = line.split("=",1)[1].strip()
```

Better would be if the python script is migrated to use [Boto3 Configuration](http://boto3.readthedocs.io/en/latest/guide/configuration.html)

However - As an alternative, I suggest the use of the following Terraform configuration: https://github.com/honestbee/loggly-elb-infra

This configuration is more robust and easier customisable for end users.

**Comment by mostlyjason** [comment]
Thanks for posting! We also have a new S3 ingestion feature coming out soon that will ingest logs directly in the product without the need for a separate script. Stay tuned!

**Comment by MichalChomo** [comment]
This is fixed by #125 .
